dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/publish/servers/crossFeatureOpenTracingZipkinServer/server.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<server description="testServer1">
+
+    <!-- Enable features -->
+    <featureManager>
+        <feature>mpOpenTracing-3.0</feature>
+        <feature>mpRestClient-3.0</feature>
+        <feature>timedexit-1.0</feature>
+        <feature>usr:opentracingZipkin-0.33</feature>
+    </featureManager>
+    
+    <opentracingZipkin host="${env.zipkinHostName}" port="${env.zipkinPortName}"/>
+    
+    <include location="../fatTestPorts.xml" />
+    
+    <!-- Automatically expand WAR files and EAR files -->
+    <applicationManager autoExpand="true" />
+
+    <webApplication id="spanTest" location="crossFeature.war" >
+        <classloader apiTypeVisibility="+third-party"/>
+    </webApplication>
+    
+    <!-- The Jaeger tracer needs a bunch of permissions. -->
+    <!-- We could scope this more tightly, but I haven't bothered because -->
+    <!-- we're not really trying to test the jaeger tracer in this test, -->
+    <!-- we're just testing compatibility with mpTelemetry -->
+    <javaPermission className="java.security.AllPermission"/>
```

Comment:
Comment says you're using the Jaeger tracer, but in this test you aren't.

The zipkin tracer _may_ need the same permissions or it may not. Try it without this line and then either remove the line or update the comment.